src/main/java/games/strategy/triplea/ui/ProductionPanel.java

```diff
@@ -82,20 +82,11 @@ protected ProductionPanel(final IUIContext uiContext) {
    */
   public IntegerMap<ProductionRule> show(final PlayerID id, final JFrame parent, final GameData data, final boolean bid,
       final IntegerMap<ProductionRule> initialPurchase) {
-    if (parent != null) {
```

Comment:
` if (parent != null) {` is being removed because it, as far as we know, is always true?